### PR TITLE
Features branch

### PR DESCRIPTION
**Pull request by nurislam-ab**
A clone of a New York Times article page. The page was built using:
- Embedded images and YouTube video
- Working with forms (buttons in this case)
- Floats and positioning
- The box model

### CODE REVIEW

**Review by fatymahmed — CHANGES_REQUESTED** [review]
## Hi @nurislam-ab, @canriquez,
You've done a great job on this project, I like how you've used the footer and aside tags 👏👏, however, there are a couple of things that you need to change.

- Your navbar doesn't seem the same as the one on the new york times original page. See the screenshot below for your navbar and what is expected.

This is what you have

Screenshot from 2019-12-11 08-47-09

This is what is expected
Screenshot from 2019-12-11 08-46-55
 
 Comments: It’s better to delete the commented code. If you need to review a previous version of your code, use git history (
no_inline_comments
see screenshot from Pathwright)

### Kindly make the suggested changes and submit for another code review.

## Status: Changes Required
_Feel free to reach out to me on slack (fatima), I'd be happy to help_

File: assets/stylesheets/main.css [hunk 1]
```diff
@@ -0,0 +1,948 @@
...
+  /* max-width: 670px; */
+}
+
+.brief-info h1 {
+  font-style: italic;
+  font-size: 2.5rem;
+  line-height: 3rem;
+  color: #121212;
+  font-weight: 700;
+
```
Comment (fatymahmed) [inline assets/stylesheets/main.css]: - Please delete any commented code as explained above

File: assets/stylesheets/main.css [hunk 2]
```diff
@@ -0,0 +1,948 @@
...
+/* -------------------------------------------
+            ARTICLE IMG & AUTHOR INFO
+ --------------------------------------------- */
+
+.brief-info {
+  margin: auto;
+  margin-top: 42px;
+  text-align: center;
+
+  /* max-width: 670px; */
```
Comment (fatymahmed) [inline assets/stylesheets/main.css]: - Please delete any commented code as explained above

File: assets/stylesheets/main.css [hunk 3]
```diff
@@ -0,0 +1,948 @@
...
+  font-style: italic;
+  font-size: 2.5rem;
+  line-height: 3rem;
+  color: #121212;
+  font-weight: 700;
+
+  /* margin-bottom:1rem; */
+  margin: 22px auto;
+
+  /* width: 670px;  */
```
Comment (fatymahmed) [inline assets/stylesheets/main.css]: - Please delete the commented code as explained above.

File: assets/stylesheets/main.css [hunk 4]
```diff
@@ -0,0 +1,948 @@
...
+
+  /* text-align: left; */
+  margin: auto;
+}
+
+/* @media(min-width:1024px){
+    .brief-info h1{
+    width:600px;
+    }
+} */
```
Comment (fatymahmed) [inline assets/stylesheets/main.css]: - You've got more commented out code,please delete these.

File: assets/stylesheets/main.css [hunk 5]
```diff
@@ -0,0 +1,948 @@
...
+  max-width: 1170px;
+
+  /* Here to control Side bar  */
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 10px;
+
+  /* padding-left: 200px;
+    padding-right: 70px; */
+}
```
Comment (fatymahmed) [inline assets/stylesheets/main.css]: - Please delete the commented code as explained above

**Review by nikasulo — CHANGES_REQUESTED** [review]
# Really great job @nurislam-ab and @canriquez! 
Your clone is exquisite and I'd really love to get it approved, however, as the last TSE rightly pointed out, there is a little issue with your navigation. Please see below:

##### What you have
screenshot

##### What is required
screenshot

Please see the requirements below for some more insight into why this is being flagged:
screenshot

# Summary
You're clone is very nice and you are both great developers! :smile: Please look into the small issue I raised above and your all set :rocket: 

If you think I missed something, or you need help or you would just like to chat, please reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilareolusakin), I will be happy to speak with you!

**Comment by canriquez** [comment]
> # Really great job @nurislam-ab and @canriquez!
> Your clone is exquisite and I'd really love to get it approved, however, as the last TSE rightly pointed out, there is a little issue with your navigation. Please see below:
> 
> ##### What you have
> screenshot
> 
> ##### What is required
> screenshot
> 
> Please see the requirements below for some more insight into why this is being flagged:
> 
> # Summary
> You're clone is very nice and you are both great developers! 😄 Please look into the small issue I raised above and your all set 🚀
> 
> If you think I missed something, or you need help or you would just like to chat, please reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [Instagram](https://instagram.com/oluwadamilareolusakin), I will be happy to speak with you!

Hi, Please check the last version of the page. We now have the right navbar as recommended by Fatima. The picture that you are including in your message contains the previous navbar version.

**Review by nikasulo — APPROVED** [review]
# Fantastico! 
You may merge :handshake:. See you at the top :rocket:

**Pull request merged**